lib/models/academic_term.dart

```diff
@@ -0,0 +1,43 @@
+///Model for Academic term
+///@author Praneet Singh
+
+import 'package:cognito/models/class.dart';
+import 'package:cognito/models/club.dart';
+
+class AcademicTerm{
+
+    DateTime startTime;
+    DateTime endTime;
+
+    Set<Class> classes;
+    Set<Club> clubs;
+
+    AcademicTerm(this.startTime, this.endTime){
+      classes = Set();
+      clubs = Set();
+    }
+
+    void setStartDate(DateTime start){
```

Comment:
Since the properties are public, we can access them directly without having to go through setters--but let's keep them for now in case we want to make them private down the line.